dashboard/test/ui/features/pd/teacher_application_dashboard.feature

```diff
@@ -11,8 +11,10 @@ Feature: Teacher Application Dashboard view
     Then I click selector "table#summary-csd-teachers ~ .btn:contains(View all applications)"
     Then I wait until element "h2:contains('CS Discoveries Teacher Applications')" is visible
     Then I wait until element "table#quick-view" is visible
-    And I open my eyes to test "Teacher Application Dashboard" except in circle
-    And I see no difference for "Teacher List View" except in circle
+
+    # TODO (mehal): Re-enable these when the test is fixed for this
+    #And I open my eyes to test "Teacher Application Dashboard" except in circle
```

Comment:
We need this line so the other "I see no difference" [steps](https://github.com/code-dot-org/code-dot-org/blob/8bde0762b08e58868cac2c166f4d007fd65b55bc/dashboard/test/ui/features/pd/teacher_application_dashboard.feature#L24) work